app/models/journeys/base.rb

```diff
@@ -18,14 +18,18 @@ def slug_sequence
       self::SlugSequence
     end
 
-    # TODO: make this work for journey specific forms
-    # that list of forms should be defined in the specific journey
     def form(claim:, params:)
-      form = SHARED_FORMS[params[:slug]]
+      avaliable_forms = SHARED_FORMS.merge(forms)
+
+      form = avaliable_forms[params[:slug]]
 
       form&.new(journey: self, claim: claim, params: params)
     end
 
+    def forms
+      {}
+    end
```

Comment:
I did something slightly different in my PR and decided that this method would return the content of `FORMS` from a specific journey model, but overriding the method is also a valid approach

Reply:
Yeah I'll rebase this once your pr is merged and pick up the change you made to `persisted?` too